Sources/Clappr/Classes/Base/Container.swift

```diff
@@ -3,6 +3,7 @@ import Foundation
 open class Container: UIObject {
     private(set) var plugins: [Plugin] = []
     @objc open var sharedData = SharedData()
+    @objc open var shouldReload = false
```

Comment:
Where this variable is being used as control?

Reply:
This is a way to force a reload on Container. You can use this reading on a `PlayButtonPlugin`, for example. What do you think?

Reply:
There is no reason to expose `Container`'s internal state and break its encapsulation to share data among plugins. The design is wrong here. We should rethink it once the only entity that should have this knowledge is `Container` itself.

Reply:
Fixed